doc/source/whatsnew/v1.4.0.rst

```diff
@@ -212,6 +212,7 @@ Other enhancements
 - :meth:`.GroupBy.mean` now supports `Numba <http://numba.pydata.org/>`_ execution with the ``engine`` keyword (:issue:`43731`)
 - :meth:`Timestamp.isoformat`, now handles the ``timespec`` argument from the base :class:``datetime`` class (:issue:`26131`)
 - :meth:`NaT.to_numpy` ``dtype`` argument is now respected, so ``np.timedelta64`` can be returned (:issue:`44460`)
+- Change in the order of the arguments in :meth:`DataFrame.to_string` due to change in the docstring (:issue:`44304`)
```

Comment:
This should go under `Other API changes` and mention that `min_rows` was affected.

Reply:
You are correct! Updated it, waiting for tests to run, surprisingly some workflows either got cancelled or failed for no obvious reasons, will look into it

Reply:
Some tests are breaking but it is not related to my code, have seen these failed in other PRs 
```
FAILED pandas/tests/plotting/test_hist_method.py::TestDataFrameGroupByPlots::test_axis_share_x
FAILED pandas/tests/plotting/test_hist_method.py::TestDataFrameGroupByPlots::test_axis_share_y
FAILED pandas/tests/plotting/test_hist_method.py::TestDataFrameGroupByPlots::test_axis_share_xy
FAILED pandas/tests/plotting/test_series.py::TestSeriesPlots::test_area_sharey_dont_overwrite
FAILED pandas/tests/plotting/frame/test_frame.py::TestDataFramePlots::test_area_sharey_dont_overwrite
FAILED pandas/tests/plotting/frame/test_hist_box_by.py::TestHistWithBy::test_axis_share_x_with_by
FAILED pandas/tests/plotting/frame/test_hist_box_by.py::TestHistWithBy::test_axis_share_y_with_by
```

Reply:
Bugs in the CI Docs and web related to #44494, no idea why but all the recent PRs have this error